spec/requests/api/moves_controller_update_spec.rb

```diff
@@ -129,7 +129,7 @@
           expect(response_json).to eq resource_to_json
         end
 
-        context 'when changing a moves documents', :skip_before do
+        context 'when changing a moves profile documents', :skip_before do
```

Comment:
still infamouse :skip_before 😱 
I guess we got not time for now to fix it

Reply:
We'll be replacing v1 moves with v2 moves soon so this will die.